### PR TITLE
Adding Twitter Scrapper

### PR DESCRIPTION
**Pull request by Guilhermeslucas**

### CODE REVIEW

File: API Google Analytics/HelloAnalytics.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+import httplib2
+from oauth2client import client
+from oauth2client import file
+from oauth2client import tools
+
+SCOPES = ['https://www.googleapis.com/auth/analytics.readonly']
+DISCOVERY_URI = ('https://analyticsreporting.googleapis.com/$discovery/rest')
+KEY_FILE_LOCATION = 'client_secrets.p12'
+SERVICE_ACCOUNT_EMAIL = 'teste-491@peerless-text-99420.iam.gserviceaccount.com'
```
Comment (hgpadovani) [inline API Google Analytics/HelloAnalytics.py:16]: I think it's better to remove this

File: API Google Analytics/HelloAnalytics.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+VIEW_ID = '3220009'
```
Comment (hgpadovani) [inline API Google Analytics/HelloAnalytics.py:17]: and this

File: API Google Analytics/HelloAnalytics.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+  return analytics
+
+def get_report(analytics):
+  # Use the Analytics Service Object to query the Analytics Reporting API V4.
+	return analytics.reports().batchGet(
+      body={
+  "reportRequests": [
+    {
+      "viewId": "3220009",
```
Comment (hgpadovani) [inline API Google Analytics/HelloAnalytics.py:42]: This viewId also must be removed

File: API_Twitter/scrapper.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    file_name (str):
+        this will be the name of the file that will be saved
+
+    Returns:
+    -------
+    true (bool):
+        this function returns True if everything went fine
+    exception (obj):
+        raises an exception if something went wrong
+    """
```
Comment (hgpadovani) [inline API_Twitter/scrapper.py:24]: Great documentation, clear and precise

**Review by hgpadovani — COMMENTED** [review]
There is not much to add, the code is really good, with great documentation and great modularization,

**Comment by Guilhermeslucas** [comment]
This stuff you commented will not enter the repository, since you modified it later, AFAIK. Try merging it and we'll see.

**Pull request merged**